packages/tools/client-debugger/client-debugger-view/src/components/ContainerSelectionDropdown.tsx

```diff
@@ -0,0 +1,80 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { Dropdown, IDropdownOption, IDropdownStyles, IStackTokens, Stack } from "@fluentui/react";
+import React from "react";
+
+import { IFluidClientDebugger } from "@fluid-tools/client-debugger";
+import { HasClientDebuggers, HasContainerId } from "../CommonProps";
+
+/**
+ * {@link ContainerSelectionDropdownProps} input props.
+ *
+ * @internal
```

Comment:
Nit: so long as we don't export this in the root `index.ts` file of this package, we don't need the `@internal` flag here.

Reply:
Removed